packages/synchro-charts/src/components/sc-dial/sc-dial-base/sc-dial-svg.tsx

```diff
@@ -0,0 +1,118 @@
+import { Component, h, Prop } from '@stencil/core';
+import { NO_VALUE_PRESENT } from '../../../constants';
+import { DataPoint, DataStream, DialSizeConfig } from '../../../utils/dataTypes';
```

Comment:
DialSizeConfig is dial specific and should be contained within the dial component folder `sc-dial`

Reply:
Done --- Move ‘DialSizeConfig’ in utils/dataTypes to sc-dial/type.ts